src/formatDate.js

```diff
@@ -51,6 +51,43 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newDate = [];
+  const oldDate = date.split(fromFormat[fromFormat.length - 1]);
+
+  function findSameElement(index) {
+    for (const oldElement of fromFormat) {
+      const indexOfOld = fromFormat.indexOf(oldElement);
+      let newElement = oldDate[indexOfOld];
+
+      if (oldElement === toFormat[index]) {
```

Comment:
with **for** (let i = 0; i < fromFormat - 1; i++)
there will be no need to use indexOf

indexOfOld === i
oldElement === fromFormat[i]
newElement === oldDate[i]

and fromFormat[i] === toFormat[index] looks clearer